### ISSUE TITLE
Specifying --cips is mandatory when cloning VM

### ISSUE THREAD

**Issue by martinmosegaard** [chef-partners/knife-vsphere#288]
### What are you trying to do and what's not working?
I want to clone a VM without any guest customization, but I get an error if I do not pass a `--cips dhcp` argument.

### Please paste the command and output (run with `-VV`). Don't forget to X out any passwords or sensitive information
```
knife vsphere -VV vm clone <name> --template win-2012-template --folder <folder> --linked-clone --resource-pool <rp> --disable-customization
...
DEBUG: value for config item customization_macs: auto
DEBUG: value for config item customization_ips:
Must specify IP numbers with --cips when specifying MAC addresses with --cmacs, can use "dhcp" as placeholder
```

### Other details, such as gem version and if you're working with a Linux or Windows VM
Gem version 1.2.14.

The error above disappears if I add `--cips dhcp` to the command line. 

In `vsphere_vm_clone.rb`, in the method `generate_clone_spec`, we build a `mac_list` for guest customization, and that uses the IP configuration. But the `mac_list` is only used with the arguments `customization_vlan` and `customization_ips`. So, if I don't want to do any of this, it would be great to just omit the `--cips` argument from the command line.

**Comment by swalberg** [comment]
Hi Martin, this is a bug. It's fixed in #278 that I think will land today (Eastern time). You could also revert to Gem version 1.2.13 to get around it.

**Comment by martinmosegaard** [comment]
Thanks, I will close this issue then!
